docs/reduction-in-locks.md

```diff
@@ -0,0 +1,106 @@
+# Reduced backup lock time
+
+--8<--- "pro-build-announcement.md"
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](#how-the---lock-ddlreduced-option-works) option to reduce the time the server remains locked by `xtrabackup` during full and incremental backups. Now, you can execute `Data Definition Language` (DDL) operations while the backup is in progress.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option key features are as follows:
+
+* Rather than holding the backup lock throughout the entire backup process, the lock is only acquired for a very short period.
+
+* DDL operations are allowed on the server while the backup is in progress.
+
+* The redo log tracks all file operations to maintain consistency. The backup process creates new metadata files to ensure data remains intact. When you run `--prepare`, the system first processes these metadata files to create a consistent database state.
+
+### Limitations
+
+* Disk read and write operations are performed during the execution of `DDL` operations and while backing up data using `xtrabackup`. Both types of operations involve accessing and modifying the data stored on disk, which can be resource-intensive.
```

Comment:
Some of the DDLs might be resource-intensive, and xtrabackup also does lot of IO to copy and read files. There could be more demand for resources from DDLs that happen during the backup.

Something one these lines..

Reply:
Certain DDL operations consume significant resources, and xtrabackup simultaneously requires I/O to copy and read files. This requirement can increase resource demand from DDL operations during the backup process.

Reply:
Thanks